code/modules/mob/living/living_defines.dm

```diff
@@ -40,6 +40,7 @@
 	var/tod = null // Time of death
 	var/update_slimes = 1
 	var/silent = null 		// Can't talk. Value goes down every life proc.
+	var/noradio = null // Softer form of silence - can't use radios, but can talk.

```

Comment:
I'm sure this can be handled in a different way. Like a CE_ effect.

Reply:
Like this?

Reply:
yep